src/EventStore.Core/StorageTracker.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using EventStore.Core.TransactionLog.LogRecords;
+
+namespace EventStore.Core;
+
+public class StorageTracker : IStorageTracker {
+	private readonly Counter<long> _readBytes;
+	private readonly Counter<long> _readEvents;
+	private readonly Counter<long> _indexedEvents;
+
+	public StorageTracker(Meter meter) {
```

Comment:
generally i've passed in a 'Metric' rather than a 'Meter' to the trackers, where a Metric is a wrapper around the instrument... In this case I think the metric would be sufficiently trivial to leave it out, but i'd still prefer to pass the instruments into the tracker rather than having the tracker create them, so that we can see all the instruments that we are creating in the bootstrapper itself